GiniVision/Classes/OnboardingPage.swift

```diff
@@ -18,6 +18,7 @@ import UIKit
     fileprivate var contentView = UIView()
     fileprivate var imageView = UIImageView()
     fileprivate var textLabel = UILabel()
+    fileprivate var needsToRotateImageInLandscape: Bool = false
```

Comment:
Don't really understand that? Why would it NOT rotate?

Reply:
The imageView is not rotated but the image itself is.